docs/get-started/02-quadratic.md

```diff
@@ -0,0 +1,82 @@
+
+The quadratic aspect of the system ensures that the cost of allocating additional credits increases quadratically, which 
+helps prevent a small number of voters from dominating the decision-making process. Quadratic voting aims to promote a 
+more accurate representation of individual preferences and allocate resources based on the collective intensity of 
+preferences rather than just a simple majority rule.
+
+See:
+
+- [Complete example](https://github.com/vocdoni/vocdoni-sdk/blob/main/examples/typescript/src/quadratic.ts)
+- [Ballot protocol implementation](https://developer.vocdoni.io/protocol/ballot#quadratic-voting)
+
+## Setting up the election
+
+Here we are going to describe how to configure an election to set up a quadratic voting. When the election object is 
+created,
+
+```ts
+const election_opts: IVoteType = {
+  maxValue: 0, // Must be 0 for quadratic elections
+  maxCount: CHOICES_LENGTH, // Number of choices for the question
+  uniqueChoices: false,
+  costFromWeight: true, // This will set that the weight on a weighted census is actually the credits available
+  costExponent: 2, // In the context of quadratic voting, this value is usually set to 2
+};
+```
+
+- The actual Vocdoni implementation only allow quadratic elections with only one question. For that, the `maxCount` 
+refers to the choices length for this unique question.  
+- `costExponent` define the exponent for quadratic voting calculation. In the context of quadratic voting, this valueis 
+usually set to 2. This exponent determines the cost of a vote. For example, if a voter assigns 2 credits to an option, 
+the cost of that vote would be 2^2 = 4. Look at SDK docs for more info.
+
+### Voter credits
+
+To determine the number of credits each voter can use, the configuration must be defined in one of two ways:
+
+1. Adjusting the `maxTotalCost`, on the election options, and using a `PlainCensus` allows for an equal number of credits to be 
+allocated to each voter.
+
+2. Alternatively, by enabling `costFromWeight` and using a `WeightedCensus`, the number of credits available will 
+correspond to the voter's weight.
+
+```ts
+  const census = new WeightedCensus();
+  census.add(
+    participants.map((participant, index) => ({
+      key: participant.address,
+      weight: BigInt(20), // Give 20 credits to every voter
+    }))
+  );
+```
+
+Using this method will enable the possibility to give different number of credits for every voter. 
+
+## Casting a vote
+
+As of now, the [protocol](https://docs.vocdoni.io/architecture/data-schemes/ballot-protocol.html#vocdoni-results-interpretation) 
+for quadratic elections only supports single question elections with discrete values. 
+
+Therefore, to cast a vote, you can use the following method:
+
+```ts
+client.submitVote(new Vote([2, 3]));
+```
+
+This approach calculates the cost of each vote based on the costExponent, for example, allocating 2 credits for option 0 
+and 3 credits for option 1:
```

Comment:
I think that this can be improved, bc we are mixing "votes" with "credit cost" IMO.  It should read something like:  "for example, allocating 2 votes for option 0 will consume 4 credits (2^2 = 4 credits) and 3 votes for option 1 will consume 9 credits (3^2 = 9 credits). 

The formula (in this case) is like:
1² = 1
2² = 4
3² = 9
4² = 16
... and so one.

Also it would be nice to know what happens if you spend less/more credits that the assigned, like: "If the voter doesn't consume all the credits available, those are lost. If the voter tries to allocate more credits than the assigned a XX error will appear."

Reply:
Well, afair if you spend more credits than the available the error thrown is not shown on the SDK (but shown on the vocdoni node). 

We should look at this too

Reply:
ed2ecb1 fixed this and added results interpretation